src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/EnumWithFlagsAttributeTests.cs

```diff
@@ -278,8 +278,23 @@ public enum MultipleMissingPowerOfTwoEnumClass
     Two = 2,
     Four = 4,
     ThirtyTwo = 32
-}}
+}}";
+
+            // Verify CA1027: Mark enums with FlagsAttribute
```

Comment:
FxCop neither reports CA1027 nor CA2217 for `MissingPowerOfTwoEnumClass` case. However, I feel your fix is appropriate as the documentation indicates CA1027 should be reported.